purple-pill/js/metapolatorStandAlone.js

```diff
@@ -39713,10 +39967,6 @@ function (
                  : resolve()
                  ;
         }
-      , cpsGenerators: {
```

Comment:
I first had the `cpsGenerators` exported at a different location as where they are now. Maybe this is some kind of artifact from that. 
The current situation is here: https://github.com/metapolator/metapolator/blob/master/app/lib/metapolatorStandAlone.js#L179

Looks like below in line 39722
```js
exports.cpsGenerators = {
   ...
```

This diff is bad to judge what happened overall, because so much is cutted out.